lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1193,6 +1189,11 @@ sub register_assets_from_settings {
     # check assets and fix the file names
     for my $k (keys %assets) {
         my $a = $assets{$k};
+        if ($a->{name} =~ /\//) {
+            log_debug "not registering asset $a->{name}";
```

Comment:
Sure, I am fine with that. I would just prefer that the message outputs a little bit about the reason. If you just read "not registering" you are going to ask "ok, why?". If the message tells "name contains invalid character, not registering." you can still think about why the specific character is invalid but at least you can immediately cope with that by fixing the name when scheduling.